client_portal/template/directory/directory.cue

```diff
@@ -23,13 +23,13 @@ roleClassList: [...directoryv1.#RoleClass] & [
 roleMap: {[string]: directoryv1.#Role} & {
 	Admin: directoryv1.#Role & {
 		id: {{ generateApplicationUUID "AdminRole" | quote }}
-		name:        "Admin"
+		name:        "AdminCSP"
```

Comment:
CSP?

Reply:
ah, I thought it's customer portal, but it's client portal.

Reply:
@jfeltesse updated to `CP` please check